coalib/parsing/LineParser.py

```diff
@@ -78,6 +79,10 @@ def _parse(self, line):
                      [(section_override, key), ...], value, to_append (True if
                      append delimiter is found else False), comment
         """
+        if re.match('[^ ]#', line) is not None or\
```

Comment:
`is not None` isn't required :) Just a `if re.match(...) or re.match(...):` will do :)